src/main/java/com/github/fishio/PlayerFish.java

```diff
@@ -396,7 +388,7 @@ public void setDead() {
 	 * Adds a life.
 	 */
 	public void addLife() {
-		lives.set(Math.min(lives.get() + 1, MAX_LIVES));
+		lives.set(Math.min(lives.get() + 1, (int) settings.getInteger("MAX_LIVES")));
```

Comment:
Don't cast to int, use Integer.intValue() (settings.getInteger("...").intValue()). Do note that if the setting is null, both ways will give problems.

<organisation